### PR TITLE
Update UI pages

### PR DESCRIPTION
**Pull request by cosletmihai**
removed `dataset_url` form forms

### CODE REVIEW

File: validator/entrypoints/ui/api_wrapper.py [hunk 1]
```diff
@@ -14,39 +14,32 @@
 from validator.entrypoints.ui import config
 
 
-def validate_file(dataset_uri: str, data_file: FileStorage, schema_file: FileStorage) -> tuple:
+def validate_file(data_file: FileStorage, schema_file: FileStorage) -> tuple:
```
Comment (costezki) [inline validator/entrypoints/ui/api_wrapper.py:17]: simple and straightforward. 
Can we have multiple schema files?

File: validator/entrypoints/ui/api_wrapper.py [hunk 2]
```diff
@@ -14,39 +14,32 @@
...
         'schema_file': (schema_file.filename, schema_file.stream, schema_file.mimetype)
     }
 
-    response = requests.post(config.VALIDATOR_API_ENDPOINT + '/validate-file', data=data, files=files)
+    response = requests.post(config.VALIDATOR_API_ENDPOINT + '/validate-file', files=files)
     return response.content, response.status_code
 
 
-def validate_sparql_endpoint(dataset_uri: str, sparql_endpoint_url: str, schema_file: FileStorage, graphs: list = None):
+def validate_sparql_endpoint(sparql_endpoint_url: str, schema_file: FileStorage, graphs: list = None):
```
Comment (costezki) [inline validator/entrypoints/ui/api_wrapper.py:34]: same here. can we provide multiple files?

File: validator/entrypoints/ui/config.py [hunk 3]
```diff
@@ -11,4 +11,4 @@
 import os
 
 FLASK_SECRET_KEY = os.environ.get('SECRET_KEY', 'secret key')
-VALIDATOR_API_ENDPOINT = os.environ.get('VALIDATOR_API_ENDPOINT', 'http://localhost:3040')
+VALIDATOR_API_ENDPOINT = os.environ.get('VALIDATOR_API_ENDPOINT', 'http://validator-api:3040')
```
Comment (costezki) [inline validator/entrypoints/ui/config.py:14]: why do we have `validator-api` as the hostname?

**Review by costezki — APPROVED** [review]
well done

File: validator/entrypoints/ui/api_wrapper.py (hunk 1 above)
Comment (cosletmihai) [reply validator/entrypoints/ui/api_wrapper.py:17]: we will, it's the next task in the pipeline

File: validator/entrypoints/ui/config.py (hunk 3 above)
Comment (cosletmihai) [reply validator/entrypoints/ui/config.py:14]: it's for development purposes. containers don't address each other by using `localohost` but the container name (it's the default docker network configuration)

**Pull request merged**